signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1791,6 +1784,672 @@ async fn sign_bitcoin_transaction() {
+        sqlx::query(
+            r#"
+            CREATE TABLE dkg_shares_copy AS
+            SELECT * FROM dkg_shares;
+            "#,
+        )
+        .execute(db.pool())
+        .await
+        .unwrap();
+
+        sqlx::query("TRUNCATE TABLE dkg_shares;")
+            .execute(db.pool())
+            .await
+            .unwrap();
+    }
+
+    // After the next bitcoin block, each of the signers will think that
+    // DKG beeds to be run. So we need to wait for it.
+    let chain_tip: BitcoinBlockHash = faucet.generate_blocks(1).pop().unwrap().into();
+
+    // We first need to wait for bitcoin-core to send us all the
+    // notifications so that we are up-to-date with the chain tip.
+    let db_update_futs = signers
+        .iter()
+        .map(|(_, db, _, _)| testing::storage::wait_for_chain_tip(db, chain_tip));
+    futures::future::join_all(db_update_futs).await;
+
+    // Now we wait for DKG to successfully complete. For that we just watch
+    // the dkg_shares table.
+    let dkg_futs = signers
+        .iter()
+        .map(|(_, db, _, _)| testing::storage::wait_for_dkg(db));
+    futures::future::join_all(dkg_futs).await;
+    let (_, db, _, _) = signers.first().unwrap();
+    let shares2 = db.get_latest_encrypted_dkg_shares().await.unwrap().unwrap();
+
+    // Time to "reveal" the DKG shares for each of the signers.
+    for (_, db, _, _) in signers.iter() {
+        sqlx::query(
+            r#"
+            INSERT INTO dkg_shares
+            SELECT * FROM dkg_shares_copy;
+            "#,
+        )
+        .execute(db.pool())
+        .await
+        .unwrap();
+
+        let dkg_share_count: i64 = sqlx::query_scalar("SELECT COUNT(*) FROM dkg_shares;")
+            .fetch_one(db.pool())
+            .await
+            .unwrap();
+
+        assert_eq!(dkg_share_count, 2);
+    }
+
+    // =========================================================================
+    // Step 10 - Prepare for 2 more deposits
+    // -------------------------------------------------------------------------
+    // - Let's create a new depositor. He will make a deposit that is
+    //   locked by the new signer aggregate key. We also make a deposit
+    //   from the old deposit, and they happen to use the old aggregate
+    //   key.
+    // - Give the new "depositor" some UTXOs so that they can make a
+    //   deposit for sBTC.
+    // =========================================================================
+    let depositor2 = Recipient::new(AddressType::P2tr);
+
+    // Start off with some initial UTXOs to work with.
+    faucet.send_to(50_000_000, &depositor2.address);
+    faucet.generate_blocks(1);
+
+    wait_for_signers(&signers).await;
+
+    // =========================================================================
+    // Step 11 - Make two proper deposits
+    // -------------------------------------------------------------------------
+    // - Use the UTXOs confirmed in steps (5) and (10) to construct two
```

Comment:
Might add that they're using both the old and new

Reply:
Updated the comment in https://github.com/stacks-network/sbtc/pull/1210/commits/d07ba0727d592bb3442411dd1a002221ca720781